readme.md

```diff
@@ -2,12 +2,71 @@
 
 This is the CSS framework for the Fusion News Theme. [Theme Blocks](https://github.com/WPMedia/fusion-news-theme-blocks) and the [Engine Theme SDK](https://github.com/WPMedia/engine-theme-sdk) import the SCSS files. Then they are [compiled](https://github.com/WPMedia/fusion/blob/e497a3117912ea3dc5ad2d0a6b83a45c2210513e/engine/webpack/_shared/rules/sass.js) later with node-sass and extracted by Webpack in Fusion.
 
+## Files Summary 
+
+As far as I can gather, the following files are included in the News Theme CSS:
```

Comment:
I think we should ensure we are using the correct voice here - rather than "As far as I can gather" - we should use working that shows we understand and own what is happening in this repo